tests/util/test_linearizer.py

```diff
@@ -119,15 +119,15 @@ def test_linearizer_is_queued(self):
         unblock2()
         self.assertFalse(linearizer.is_queued(key))
 
-    def test_lots_of_queued_things(self):
+    def test_lots_of_queued_things(self) -> None:
         """Tests lots of fast things queued up behind a slow thing.
 
         The stack should *not* explode when the fast thing completes.
         """
         linearizer = Linearizer()
         key = ""
 
-        async def func(i, wait_for=None) -> None:
+        async def func(i: int, wait_for: Optional["Deferred[None]"] = None) -> None:
```

Comment:
is (was?) `wait_for` unused?

Reply:
it's used in the very first call (the "slow thing")

Reply:
On -92 I can see `func(0, sleep=True)`, but I can't see an analogous call to func after the changes.

Maybe the point is that the slow thing is now:
- acquiring the linearizer lock on +139
- "waiting" to release it on +144?

If so, I still think `wait_for` is unused?

Reply:
```suggestion
        async def func(i: int) -> None:
```

Reply:
It's gone now!